### ISSUE TITLE
ARM64

### ISSUE THREAD

**Issue by parish99** [maxtacu/echo-speaks-docker#4]
Maxim, it looks like when you rebuilt this you used AMD instead of ARM this time?

**Comment by maxtacu** [comment]
Yep. I will build 2 separate images (for amd and arm) and add them with appropriate tag.

**Comment by j-parish** [comment]
Did you ever get around to re-making the ARM 64 image for this?
